packages/common/src/utils/dateUtils.test.ts

```diff
@@ -63,17 +64,88 @@ const configList: ConfigUeCheckboxDateRange[] = [
 describe('Date utils tests', () => {
   it('get valid date with valid dashes string', () => {
     const validDateStringDashes = '2021-04-08'
-    const date = getValidDate(validDateStringDashes)
+    const date = getValidDateFormat(validDateStringDashes)
     expect(date).toBeTruthy()
   })
 
   it('get valid date without dashes string', () => {
     const validDateString = '20210408'
+    const date = getValidDateFormat(validDateString)
+    // console.log(date)
+    expect(date).toBeTruthy()
```

Comment:
Would prefer to check on the actual return on this so that instead of "truthy" we check the exact output `expect(date).toEqual('20210408')` and instead of "falsy" we should check that it's undefined `expect(date).toBeUndefined()`